api/resources/views/filled-form/edit.blade.php

```diff
@@ -0,0 +1,25 @@
+@extends('layouts.app')
+
+@section('content')
+
+    <div class="container mt-4">
+        @include('errors')
+        <form action="{{ route('filled-forms.store') }}" method="post">
+            @csrf
+            @foreach($fields as $field => $status)
+                <div class="form-group">
+                    @if($statuses->get($status) !== 'unused' )
+                        <label style="text-transform: capitalize" for="{{$field}}">{{ implode(" ", explode('_', $field)) }}</label>
+                        <input type="text" class="form-control form-control" id="{{$field}}" name="{{$field}}"
+                               placeholder="Enter {{ implode(" ", explode('_', $field)) }}"
+                               value = "{{ $filledForm->$field }}">
+                    @endif
+                </div>
+            @endforeach
+
+            <input hidden style="display: none" type="text" name="formId" value="{{ $jobFormId }}">
```

Comment:
Why is this hidden AND display:none? if that's really necessary, class="d-none" instead of an inline style, perhaps?

Reply:
double hidde motherfu**er :DD jk , going to fix it

Reply:
done